.github/templates/publish-release.yml

```diff
@@ -0,0 +1,93 @@
+#@ load("@ytt:template", "template")
+
+#@ def uploadToNuget(packageName, isSymbols = False):
+#@ extension = "nupkg"
+#@ if isSymbols:
+#@   extension = "snupkg"
+#@ end
+  - name: #@ "NuGet Publish " + packageName + ".${{ steps.get-version.outputs.version }}"
+    run: #@ "dotnet nuget push ${{ github.workspace }}/Realm/packages/" + packageName + ".${{ steps.get-version.outputs.version }}/" + packageName + ".${{ steps.get-version.outputs.version }}." + extension + " --skip-duplicate --api-key ${{ secrets.NUGET_API_KEY }} --source https://api.nuget.org/v3/index.json"
+#@ end
+
+#@ def uploadToNpm(packageName):
+  - uses: actions/setup-node@v2
+    with:
+      node-version: '12.x'
+      registry-url: 'https://registry.npmjs.org'
+  - name: #@ "Npm Publish " + packageName + "-${{ steps.get-version.outputs.version }}"
+    run: #@ "npm publish ${{ github.workspace }}/Realm/packages/" + packageName + "-${{ steps.get-version.outputs.version }}.tgz/" + packageName + "-${{ steps.get-version.outputs.version }}.tgz"
+    env:
+      NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+#@ end
+
+#@ def publishGithubRelease():
+  - name: Create Github Release
+    uses: ncipollo/release-action@v1
+    with:
+      artifacts: ${{ github.workspace }}/Realm/packages/io.realm.unity-${{ steps.get-version.outputs.version }}.tgz/io.realm.unity-${{ steps.get-version.outputs.version }}.tgz
+      bodyFile: ${{ steps.extract-release-notes.outputs.release-notes-path }}
+      name: ${{ steps.get-version.outputs.version }}
+      commit: ${{ github.ref }}
+      tag: ${{ steps.get-version.outputs.version }}
+      token: ${{ secrets.GITHUB_TOKEN }}
+      draft: true
+#@ end
+
+#@ def uploadDocsToS3():
+  - name: Configure AWS Credentials
+    uses: aws-actions/configure-aws-credentials@v1
+    with:
+      aws-access-key-id: ${{ secrets.DOCS_S3_ACCESS_KEY }}
+      aws-secret-access-key: ${{ secrets.DOCS_S3_SECRET_KEY }}
+      aws-region: us-east-2
+  - name: Upload docs
+    run: |
+      Expand-Archive -Path Realm/packages/Docs.zip/Docs.zip -DestinationPath Realm/packages
+      $versions = "${{ steps.get-version.outputs.version }}", "latest"
+      Foreach ($ver in $versions)
+      {
+        aws s3 sync --acl public-read "${{ github.workspace }}\Realm\packages\_site" s3://realm-sdks/realm-sdks/dotnet/$ver/
+      }
+#@ end
+---
+name: Publish Release
+"on":
+  workflow_dispatch
+
+jobs:
+  main:
+    runs-on: windows-latest
+    environment: Production
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+      with:
+        submodules: false
+    - name: Download all artifacts
+      uses: dawidd6/action-download-artifact@v2
+      with:
+        workflow: main.yml
+        commit: ${{ github.sha }}
+        path: ${{ github.workspace }}/Realm/packages/
+    - name: Extract release notes
```

Comment:
Maybe we should put in the todo of this PR to instead fetch the `ExtractedChangelog.md` like I'm doing in [my PR](https://github.com/realm/realm-dotnet/blob/172a88b7b0b6ba9364e3b778d706f956299b8ec3/.github/workflows/release.yml#L36).

Reply:
I don't think it needs a TODO - when we merge this, it will show up as a conflict in your PR which we'll resolve like usual.